sklearn/linear_model/tests/test_ridge.py

```diff
@@ -692,6 +692,36 @@ def test_sparse_design_with_sample_weights():
                                       decimal=6)
 
 
+def test_ridgecv_alphas():
+    # Test that no error is raised when fitting RidgeCV
```

Comment:
I would remove this comment since that it is obvious from the renaming from the function